CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Add option `withDarkBorder` in `hoc/card`.
+- Feature: Change `border-color` in `hoc/card` when prop `href: true`.
```

Comment:
Ici, je pense que la modification, c'est plutôt "Add `withDarkBorder` prop when prop `href` is `true` on `ProjectCreatorCard` component."